unit-tests-jest/numbers.test.ts

```diff
@@ -0,0 +1,48 @@
+import { evenNumbers, toDollars, divideBy, multiplyBy } from './numbers';
+
+export function isIsogram(str: string): boolean {
+  for (let i = 0; i < str.length; i++) {
+    const string = str.toLowerCase().split('');
+    string.splice(i, 1);
+    if (string.join('').includes(str[i])) {
+      return false;
+    }
+  }
+  return true;
+}
+
+describe('Isogram', () => {
+  it('returns true as a isogram', () => {
+    expect(isIsogram('isogram')).toStrictEqual(true);
+  });
+});
+
+describe('evenNumbers', () => {
+  it('Returns a new array that contains only the even numbers in the given array.', () => {
+    expect(evenNumbers([1, 4, 5, 10, 0])).toStrictEqual([4, 10, 0]);
+  });
+});
+describe('toDollars', () => {
+  it('Returns a number formatted in dollars and cents.', () => {
+    expect(toDollars(3)).toBe('$3.00');
+  });
+  it('Returns a number formatted in dollars and cents.', () => {
```

Comment:
You should make sure each description is different so that when Jest reports the test result it is easy to see which one is failing.